sentences/de/lock_HassTurnOn.yaml

```
@@ -0,0 +1,14 @@
+language: de
+intents:
+  HassTurnOn:
+    data:
+      - sentences:
+          - "<sperren> <name> [<area>] [zu|ab]"
+          - "<sperren> <area> <name> [zu|ab]"
```

```suggestion
          - "(<sperren>|<machen>) <name> [<area>] [zu|ab]"
          - "(<sperren>|<machen>) <area> <name> [zu|ab]"
```
this also allows sentences like "mach die Tür im Flur zu"